Hardware/Hardware/Printer/AirPrintReceipt/AirPrintReceiptPrinterService.swift

```diff
@@ -38,3 +44,27 @@ public final class AirPrintReceiptPrinterService: PrinterService {
         }
     }
 }
+
+extension AirPrintReceiptPrinterService: UIPrintInteractionControllerDelegate {
+    public func printInteractionController(_ printInteractionController: UIPrintInteractionController, choosePaper paperList: [UIPrintPaper]) -> UIPrintPaper {
+        let pageSize = self.receiptContent?.preferredPageSizeForPrinting ?? AirPrintReceiptPrinterService.defaultReceiptPageSize
+        return UIPrintPaper.bestPaper(forPageSize: pageSize, withPapersFrom: paperList)
+    }
+
+    public func printInteractionController(_ printInteractionController: UIPrintInteractionController, cutLengthFor paper: UIPrintPaper) -> CGFloat {
+        let pageSize = self.receiptContent?.preferredPageSizeForPrinting ?? AirPrintReceiptPrinterService.defaultReceiptPageSize
+        return pageSize.height + AirPrintReceiptPrinterService.defaultRollCutterMargin
+    }
+}
+
+extension AirPrintReceiptPrinterService {
+    static let pointsPerInch: Int = 72
```

Comment:
Is this intrinsic to the printer or content? The stuff we're printing on the Brother seems pretty... big, and it seems like this might be why. That said, a 4" wide receipt is quite big.

Reply:
Apple examples include it as `POINTS_PER_INCH` but I struggled to find that in Swift - e.g. see https://devstreaming-cdn.apple.com/videos/wwdc/2014/718xxctf8ley20j/718/718_adopting_airprint.pdf

It appears the consensus is 72 for printed matter, so I went with it.